### DISCUSSION TITLE
UI Logs on 20.0.0 - raspi 2 zero W

### DISCUSSION THREAD

**Discussion by zigomatichub** [awawa-dev/HyperHDR#897] [Q&A] [answered]
After installing the new version, the part of the UI concerning Logs are empty.

I see some logs in ssh in /var/log/daemon.  How can I configure and see logs in UI please ?
I search about a .conf or a json file but I didn't see anything on my system. seems everything is in db sqlite ?

![image](https://github.com/awawa-dev/HyperHDR/assets/60198279/f343c55d-0b76-4f90-9710-4a0801b97b59)

in db, I see this 
debugLevel":"0","debugStreamer":false

**Comment by awawa-dev** [comment, marked as answer]
general tab-> Logging -> Log-Level = debug
next restart

**Comment by zigomatichub** [comment]
I found also in DB - logger{"level":"warn"} 

Ah thank you @awawa-dev . I'm sorry... It's so evident now !

**Comment by zigomatichub** [comment]
@awawa-dev 
FYI
But I got a popup saying there is an error where with the default level as Warn, nothing was visible. Now in debug, I got them all.
